packages/components/addon/components/hds/copy/button/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export const DEFAULT_SIZE = 'medium';
+export const SIZES = ['small', 'medium'];
+export const DEFAULT_ICON = 'clipboard-copy';
+export const SUCCESS_ICON = 'clipboard-checked';
+export const ERROR_ICON = 'clipboard';
+export const DEFAULT_STATUS = 'idle';
+
+export default class HdsCopyButtonComponent extends Component {
+  @tracked status = DEFAULT_STATUS;
+  @tracked timer;
+
+  /**
+   * @param textToCopy
+   * @type {string}
+   * @description The string to be copied.
+   */
+  get textToCopy() {
+    return this.args.textToCopy;
+  }
+
+  /**
+   * @param targetToCopy
+   * @type {string}
+   * @description The id of the element whose content should be copied.
+   */
+  get targetToCopy() {
+    return this.args.targetToCopy;
+  }
+
+  /**
+   * @param container
+   * @type {string}
+   * @description For cases where focus management is needed, set the focused element as the container value.
+   */
+  get container() {
```

Comment:
FWIW i don't see this documented in the docs PR

Reply:
Good catch, we only just added support for this, I'll put it in the docs now. 👍